styles/base.css

```diff
@@ -1,3 +1,10 @@
+:root {
+    --gradient1:linear-gradient(130deg, rgb(255, 177, 140) 0%, rgb(255, 218, 201) 100%);
+    --gradient2:linear-gradient(130deg, rgb(255, 218, 201) 0%, rgb(255, 177, 140) 100%);
```

Comment:
I really like the gradients! The banana one is fun, but I slightly prefer the standard one.